packages/react/src/progress/Progress.css.ts

```diff
@@ -0,0 +1,38 @@
+import { theme } from "@optiaxiom/globals";
+
+import { recipe, type RecipeVariants, style } from "../vanilla-extract";
+
+export const progress = recipe({
+  base: [
+    {
+      h: "full",
+    },
+    style({
+      transition: "all 400ms",
+    }),
```

Comment:
why custom style here instead of the transition all sprinkle like before?

Reply:
The transition didn't work expectedly before, so I added it check it vai. Going to revert it.